core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -123,3 +187,7 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
     });
   }
 }
+
+const WEB_SOCKET_HOST = "localhost";
+const PYTHON_LANGUAGE_SERVER = "/python-language-server";
+const LANGUAGE_SERVER_PORT = 3000;
```

Comment:
get those into the configuration file, i.e., `environments.ts` and `environments.prod.ts`.